Makefile

```diff
@@ -27,7 +27,7 @@ help: Makefile
 testDataDir := test/data/
 tempDir := ${testDataDir}temp/
 gitDataDir := ${tempDir}sdk-test-data/
-branchName := main
+branchName := tp/bootstrap-config
```

Comment:
I reckon you'll change this back once you've merged your bootstrap test data?

Reply:
done